.lintstagedrc.json

```diff
@@ -1,3 +1,4 @@
 {
-  "*.{js,jsx,ts,tsx,json,md}": ["npx prettier --write"]
+  "*.{js,jsx,ts,tsx}": ["npx prettier --write", "npx eslint --fix"],
+  "*.{json,md,mdx}": ["npx prettier --write"]
```

Comment:
lint-staged runs things concurrently. With this setup prettier and ESLint run after each other for js/jsx/ts/tsx files